contracts/consensus-gateway/ERC20GatewayBase.sol

```diff
@@ -80,4 +86,30 @@ contract ERC20GatewayBase {
         );
     }
 
+    /* Internal functions */
```

Comment:
Please add a new line before `/* Internal functions */`
And remove an extra newline from `L62`/`L63`.

Reply:
Done.